com.github.adrienverge.PhotoCollage.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.adrienverge.PhotoCollage
```

Comment:
id should start with `io.github`

Reply:
Really https://docs.flatpak.org/en/latest/conventions.html#application-ids

Those with `com.github` are a mistake, but they are "grandfathered"

New submissions though. The linter will no allow it.

Reply:
Alright, thank you. It's done now